docker-compose.armv7.yml

```diff
@@ -0,0 +1,113 @@
+# Use admin/pass as user/password credentials to login to openemr (from OE_USER and OE_PASS below)
+# MYSQL_HOST and MYSQL_ROOT_PASS are required for openemr
+# EASY_DEV_MODE prevents issues with permissions when mounting volumes
+# EAST_DEV_MODE_NEW with value of 'yes' expands EASY_DEV_MODE by not requiring downloading
+#  code from github (uses local repo).
+# MYSQL_USER, MYSQL_PASS, OE_USER, MYSQL_PASS are optional for openemr and
+#   if not provided, then default to openemr, openemr, admin, and pass respectively.
+version: '3.1'
+services:
+  mysql:
+    restart: always
+    image: jsurf/rpi-mariadb
+    command: ['mysqld','--character-set-server=utf8mb4','--ssl-ca=/etc/ssl/ca.pem','--ssl_cert=/etc/ssl/server-cert.pem','--ssl_key=/etc/ssl/server-key.pem']
+    ports:
+    - 8320:3306
+    volumes:
+    - ./contrib/util/docker/sql-ssl-certs-keys/easy/ca.pem:/etc/ssl/ca.pem:ro
+    - ./contrib/util/docker/sql-ssl-certs-keys/easy/server-cert.pem:/etc/ssl/server-cert.pem:ro
+    - ./contrib/util/docker/sql-ssl-certs-keys/easy/server-key.pem:/etc/ssl/server-key.pem:ro
+    - databasevolume:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+  openemr:
+    restart: always
+    # use an image if you're not changing the build steps:
+    image: jesdynf/openemr:flex-3.12
+    # if editing the Dockerfile, clone the devops repo and point to the path:
+    # build: ../openemr-devops/docker/openemr/flex-edge/
+    ports:
+    - 8300:80
+    - 9300:443
+    volumes:
+    - .:/openemr:ro
+    - .:/var/www/localhost/htdocs/openemr:rw
+    - publicvolume:/var/www/localhost/htdocs/openemr/public:rw
+    - sitesvolume:/var/www/localhost/htdocs/openemr/sites:rw
+    - nodemodules:/var/www/localhost/htdocs/openemr/node_modules:rw
+    - vendordir:/var/www/localhost/htdocs/openemr/vendor:rw
+    - ccdamodule:/var/www/localhost/htdocs/openemr/ccdaservice:rw
+    - logvolume:/var/log
+    - couchdbvolume:/couchdb/data
+    environment:
+      MYSQL_HOST: mysql
+      MYSQL_ROOT_PASS: root
+      MYSQL_USER: openemr
+      MYSQL_PASS: openemr
+      OE_USER: admin
+      OE_PASS: pass
+      EASY_DEV_MODE: "yes"
+      EASY_DEV_MODE_NEW: "yes"
+      # e2e requires chromium, which alpine isn't giving us in arm
+      DEVELOPER_TOOLS: "no"
```

Comment:
gonna need to make a setting specific to chromium/e2e in the flex series dockers otherwise none of other composer developer packages come in

Reply:
I agree but it's beyond my expertise.